boards/openmote-b/dist/debug.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+# Start in-circuit debugging on this board: this script starts up the GDB
+# client and connects to a GDB server.
+#
+# Start the GDB server first using the 'make debugserver' target
```

Comment:
I think the target is `debug-server`. With that I was able start a debug server and a client that connects but however, I never reached main() in GDB. Did you try?

Reply:
Not really.  The tui feature was removed in my version of gdb.

Reply:
I wouldn't mind if we don't set tui by default. We don't do that for most of our platforms anyway, do we?